code/modules/events/wizard/shuffle.dm

```diff
@@ -61,7 +61,9 @@
 	for(var/mob/living/carbon/human/H in mobs)
 		if(!mobnames)
 			break
-		H.real_name = mobnames[mobnames.len]
+		var/mobname = mobnames[mobnames.len]
+		H.real_name = mobname[1]
+		H.voiceprint = mobname[2]
```

Comment:
Line 52 deals with getting all the mob's name and voiceprint, and indexing them in a list, this stuff is actually giving out the shuffled names and voiceprints.